harvester/search/models/elastic.py

```diff
@@ -17,7 +17,7 @@ class ElasticIndex(models.Model):
 
     name = models.CharField(max_length=255)
     language = models.CharField(max_length=5, choices=settings.ELASTIC_SEARCH_ANALYSERS.items())
-    freeze = models.ForeignKey(Freeze, related_name="indices")
+    freeze = models.ForeignKey(Freeze, on_delete=models.SET_NULL, null=True, related_name="indices")
```

Comment:
Indices should always have a freeze. You could go for PROTECT here. That way we warn if people delete a freeze and are not cleaning up their indices in Elastic